website/content/api-docs/secret/identity/entity-alias.mdx

```diff
@@ -6,6 +6,10 @@ description: >-
   store.
 ---
 
+~> **NOTE:** Be careful in granting permissions to non-readonly identity group
```

Comment:
I couldn't tell what the word "group" meant here. Should this sentence go on the group page and be switched with the one that's currently on the group page?

Reply:
Yeah, got it backwards.